server/etcdserver/api/etcdhttp/health.go

```diff
@@ -342,6 +378,23 @@ func listToStringSet(list []string) StringSet {
 	return set
 }
 
+func recordMetrics(path, name string, status string) {
+	val := 0
+	if status == HealthStatusSuccess {
+		val = 1
+	}
+	path = strings.TrimPrefix(path, "/")
```

Comment:
```suggestion
	type = strings.TrimPrefix(path, "/")
```

Reply:
Can we change the `path` field in `CheckRegistry` into `type` to avoid trimming prefix?

Reply:
done